src/pages/task/taskTabs/buildBaronAndAnnotations/BBCreatedTickets.tsx

```diff
@@ -1,108 +1,65 @@
 import React from "react";
-import { useQuery } from "@apollo/client";
-import { useToastContext } from "context/toast";
-import {
-  GetCustomCreatedIssuesQuery,
-  GetCustomCreatedIssuesQueryVariables,
-  GetCreatedTicketsQuery,
-  GetCreatedTicketsQueryVariables,
-} from "gql/generated/types";
-import {
-  GET_CREATED_TICKETS,
-  GET_JIRA_CUSTOM_CREATED_ISSUES,
-} from "gql/queries";
-import { CustomCreatedTicketsTable } from "./AnnotationTicketsTable";
+import { JiraTicket, IssueLink } from "gql/generated/types";
 import { TicketsTitle, TitleAndButtons } from "./BBComponents";
 import { FileTicket } from "./BBFileTicket";
-import { BuildBaronTable } from "./BuildBaronTable";
 
 interface CreatedTicketsProps {
   taskId: string;
   execution: number;
   buildBaronConfigured: boolean;
+  tickets: JiraTicket[] | IssueLink[];
```

Comment:
Isn't this a bit operator? Does this combine the lists here?

Reply:
no, it can be either type. We need this because it can be either a jira ticket or a different type of ticket.